tests/test_helper.tcl

```diff
@@ -327,7 +327,7 @@ proc run_solo {name code} {
 proc cleanup {} {
     if {!$::quiet} {puts -nonewline "Cleanup: may take some time... "}
     flush stdout
-    catch {exec rm -rf {*}[glob tests/tmp/redis.conf.*]}
+    catch {exec rm -rf {*}[glob tests/tmp/placeholderkv.conf.*]}
```

Comment:
```suggestion
    catch {exec rm -rf {*}[glob tests/tmp/server.conf.*]}
```
Is there some reason this can't also be server? (Actually, this might just be silently leaking right now)